doc/test-framework/scorecard.md

```diff
@@ -325,11 +325,157 @@ Example of a valid JSON output:
+                    - name: scorecard-kubeconfig
+                      mountPath: /scorecard-secret
+                    env:
+                      # scorecard kubeconfig env
+                    - name: KUBECONFIG
+                      value: /scorecard-secret/config
+                  - name: container2
+                    # Do the same for this and all other containers.
+                    ...
+    ```
+1. Insert the scorecard proxy container into the operator's `Deployment`:
+    ```yaml
+    spec:
+      install:
+        spec:
+          deployments:
+          - name: memcached-operator
+            spec:
+              ...
+              template:
+                ...
+                spec:
+                  containers:
+                  ...
+                  # scorecard proxy container
+                  - name: scorecard-proxy
+                    command:
+                    - scorecard-proxy
+                    env:
+                    - name: WATCH_NAMESPACE
+                      valueFrom:
+                        fieldRef:
+                          apiVersion: v1
+                          fieldPath: metadata.namespace
+                    image: quay.io/operator-framework/scorecard-proxy:master
+                    imagePullPolicy: Always
+                    ports:
+                    - name: proxy
+                      containerPort: 8889
+    ```
+
+Alternatively, the [community-operators][community-operators] repo has several bash functions that can perform these operations for you:
+```sh
+$ curl -Lo csv-manifest-modifiers.sh https://raw.githubusercontent.com/operator-framework/community-operators/master/scripts/lib/file
+$ . ./csv-manifest-modifiers.sh
+# $NAMESPACE is the namespace your operator will deploy in
+$ create_kubeconfig_secret_file scorecard-secret.yaml "$NAMESPACE"
+$ kubectl apply -f scorecard-secret.yaml
+# $CSV_FILE is the path to your operator's CSV manifest
+$ insert_kubeconfig_volume "$CSV_FILE"
+$ insert_kubeconfig_secret_mount "$CSV_FILE"
+$ insert_proxy_container "$CSV_FILE" "quay.io/operator-framework/scorecard-proxy:master"
+```
 
-Running with a CSV alone requires both the `--csv-path=<CSV manifest path>` and `--olm-deployed` flags to be set. The scorecard assumes your CSV and relevant CRD's have been deployed onto the cluster using the OLM when using `--olm-deployed`. This [document][olm-deploy-operator] walks through bundling your CSV and CRD's, deploying the OLM on minikube or [OKD][okd], and deploying your operator. Once these steps have been completed, run the scorecard with both the `--csv-path=<CSV manifest path>` and `--olm-deployed` flags.
+Once done, follow the steps in this [document][olm-deploy-operator] to bundle your CSV and CRD's, deploy OLM on minikube or [OKD][okd], and deploy your operator. Once these steps have been completed, run the scorecard with both the `--csv-path=<CSV manifest path>` and `--olm-deployed` flags.
```

Comment:
It shows a little confuse to me. Maybe something as : 
```suggestion
After following the steps in this [document][olm-deploy-operator] to bundle your CSV and CRD's and have your operator deployed in the cluster with OLM run the scorecard with both the `--csv-path=<CSV manifest path>` and `--olm-deployed` flags.
```

Reply:
I prefer the original text, since it clearly states your should follow the steps in the linked document to do everything listed.